### PR TITLE
Fix thread count crash

### PR DESCRIPTION
**Pull request by rsoltani97**

### CODE REVIEW

File: src/utils.cpp [hunk 1]
```diff
@@ -837,28 +837,54 @@ bool is_left_chain(chain_t a, chain_t b) {
 // 		fprintf(stderr, "NOV\n");
         return a_beg < b_beg;
     } else {
+//        for (int i = 0; i < a.chain_len; i++)
+//            cout << "a " << a.frags[i].rpos << " " << a.frags[i].qpos << " " << a.frags[i].len << endl;
+//        for (int i = 0; i < b.chain_len; i++)
+//            cout << "b " << b.frags[i].rpos << " " << b.frags[i].qpos << " " << b.frags[i].len << endl;
+
```
Comment (HosseinAsghari) [inline src/utils.cpp]: Please remove these lines

File: src/utils.cpp [hunk 2]
```diff
@@ -837,28 +837,54 @@ bool is_left_chain(chain_t a, chain_t b) {
 // 		fprintf(stderr, "NOV\n");
         return a_beg < b_beg;
     } else {
+//        for (int i = 0; i < a.chain_len; i++)
+//            cout << "a " << a.frags[i].rpos << " " << a.frags[i].qpos << " " << a.frags[i].len << endl;
+//        for (int i = 0; i < b.chain_len; i++)
+//            cout << "b " << b.frags[i].rpos << " " << b.frags[i].qpos << " " << b.frags[i].len << endl;
+
         uint32_t i = 0, j = 0;
+        int best_distance = 1000000000;
```
Comment (HosseinAsghari) [inline src/utils.cpp]: Use INF instead of the initialized value. INF is a defined value to 10^9.

**Review by HosseinAsghari — COMMENTED** [review]
Looks good in general. I added two comments. 
Let me know when you made the changes and I'll approve the merge afterward.

**Comment by rsoltani97** [comment]
I resolved your comments. @HosseinAsghari

**Review by HosseinAsghari — APPROVED** [review]

**Pull request merged**